sql_generators/active_users_aggregates_v3/templates/focus_android_query.sql

```diff
@@ -17,11 +16,26 @@ WITH baseline AS (
     device AS device_model,
     first_seen_date,
     submission_date = first_seen_date AS is_new_profile,
-    NULL AS uri_count,
-    default_browser AS is_default_browser,
     distribution_id,
     CAST(NULL AS string) AS isp,
-    'Focus Android' AS app_name
+    'Focus Android Legacy' AS app_name,
+    CASE
```

Comment:
For Legacy, I'm proposing here to keep the code previously used to calculate segments, given that days_active_bits is not available. Currently Glean data is used for KPI reporting, this is mostly in case the aggregate is backfilled. It could also be NULL. Thoughts @bochocki?

Reply:
We don't use any Legacy mobile data for KPI reporting; this can be NULL. The "Legacy" string you're proposing works, too.